test/modules/standard/Random/deitz/test1D2D.chpl

```diff
@@ -113,8 +127,10 @@ config const n: int = 8;
   var A: [{1..n} dmapped Block(rank=1,boundingBox={1..n/2})] real;
   forall (r,a) in zip(rs.iterate(A.domain), A) do
     a = r;
-  writeln(for e in A do format("#######", e.locale.id));
-  writeln(for e in A do format("#.#####", e));
+  for e in A do writef(" #######", e.locale.id);
+  writeln();
+  for e in A do writef(" #.#####", e);
+  writeln();
```

Comment:
These tests are arguably showing the case when having format() is useful even if we have string.writef().